### PR TITLE
Update simple-acl-controlled-application.rst

### PR DESCRIPTION
**Pull request by Benuva**
The template file 'app/View/Users/login.ctp' is missing the open and close tags for PHP.

### CODE REVIEW

File: en/tutorials-and-examples/simple-acl-controlled-application/simple-acl-controlled-application.rst [hunk 1]
```diff
@@ -144,6 +145,7 @@ Then create the following view file for login at
         'password'
     ));
     echo $this->Form->end('Login');
+    ?>
```
Comment (ravage84) [inline en/tutorials-and-examples/simple-acl-controlled-application/simple-acl-controlled-application.rst]: :-1:

**Comment by dereuromark** [comment]
Looks good to me

File: en/tutorials-and-examples/simple-acl-controlled-application/simple-acl-controlled-application.rst (hunk 1 above)
Comment (markstory) [inline en/tutorials-and-examples/simple-acl-controlled-application/simple-acl-controlled-application.rst]: We generally don't include close tags.

**Comment by markstory** [comment]
Merging and removing close tag post merge.

**Pull request merged**